charts/sqs-controller-chart/templates/deployment.yaml

```diff
@@ -1,92 +1,116 @@
+      {{- range $key, $value := .Values.deployment.annotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+      labels:
+        app.kubernetes.io/name: {{ include "app.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: Helm
+        k8s-app: {{ include "app.name" . }}
+{{- range $key, $value := .Values.deployment.labels }}
+        {{ $key }}: {{ $value | quote }}
+{{- end }}
     spec:
+      serviceAccountName: {{ include "service-account.name" . }}
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end }}
+      {{- end }}
       containers:
-      - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
-        - --v=0
-        image: {{ .Values.controllerManager.kubeRbacProxy.image.repository }}:{{ .Values.controllerManager.kubeRbacProxy.image.tag
-          | default .Chart.AppVersion }}
-        name: kube-rbac-proxy
+      - command:
+        - ./bin/controller
+        args:
+        - --aws-region
+        - "$(AWS_REGION)"
+        - --aws-endpoint-url
+        - "$(AWS_ENDPOINT_URL)"
+        - --enable-development-logging
+        - "$(ACK_ENABLE_DEVELOPMENT_LOGGING)"
+        - --log-level
+        - "$(ACK_LOG_LEVEL)"
+        - --resource-tags
+        - "$(ACK_RESOURCE_TAGS)"
+        - --watch-namespace
+        - "$(ACK_WATCH_NAMESPACE)"
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        name: controller
         ports:
-        - containerPort: 8443
-          name: https
-          protocol: TCP
-        resources: {{- toYaml .Values.controllerManager.kubeRbacProxy.resources | nindent
-          10 }}
-      - args:
-        - --health-probe-bind-address=:8081
-        - --metrics-bind-address=127.0.0.1:8080
-        - --leader-elect
-        command:
-        - /manager
-        image: {{ .Values.controllerManager.manager.image.repository }}:{{ .Values.controllerManager.manager.image.tag
-          | default .Chart.AppVersion }}
+          - name: http
+            containerPort: {{ .Values.deployment.containerPort }}
+        resources:
+          {{- toYaml .Values.resources | nindent 10 }}
         env:
-          {{- if .Values.aws.region }}
-          - name: AWS_REGION
-            value: {{ .Values.aws.region }}
-          {{- end }}
-          {{- if .Values.aws.credentials.secretName }}
-          - name: AWS_SHARED_CREDENTIALS_FILE
-            value: {{ include "aws.credentials.path" . }}
-          - name: AWS_PROFILE
-            value: {{ .Values.aws.credentials.profile }}
+        - name: ACK_SYSTEM_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: AWS_REGION
+          value: {{ .Values.aws.region }}
```

Comment:
Notice the default is an empty string. Is that OK? If not, add a `required`

Reply:
no, but we always pass from the chart, so it should never happen. I'll add just in case 👍

Reply:
actually i'll add to all the controllers, its weird just one will have it.